File: fort.java
import java.util.Scanner;

/*
Solution to Fort by David Harmeyer

Samples:
3
15 10 2
0 200 10
250 100 15
 */
public class fort {
   public static void main(String[] args) {
      Scanner fs=new Scanner(System.in);

      // Loop over months
      int months=fs.nextInt(), i;
      for (i=1; i<=months; i++) {
         // Output header for month
         System.out.printf("Month #%d:%n", i);

         // Loop over collection days within this month
         int days=fs.nextInt(), w=0, b=0, m=0, d;
         for (d=1; d<=days; d++) {
            // Read stats collected and sum
            int ww=fs.nextInt(), bb=fs.nextInt(), mm=fs.nextInt();
            System.out.printf("Day #%d: Collected %d wood, %d brick, and %d metal.%n", d, ww, bb, mm);
            w+=ww;
            b+=bb;
            m+=mm;
         }

         // Output total for month
         System.out.printf("Collected a total of %d wood, %d brick, and %d metal!%n%n", w, b, m);
      }
   }
}

File: fort.cpp
#include<iostream>

using namespace std;

int main() {
   int m;

   // Read in number of months
   cin>>m;

   // Loop over each month
   int j;
   for(j = 1; j <= m; j++) {
      // Output month header
      cout << "Month #" << j << ":" << endl;

      // Read in number of days
      int n;
      cin>>n;

      // Process each day, output that day, and sum
      int i;
      int totW = 0, totB = 0, totM = 0;
      for(i = 1; i <= n; i++) {
         int curW, curB, curM;
         cin >> curW >> curB >> curM;
         cout << "Day #" << i << ": Collected " << curW << " wood, " << curB << " brick, " << "and " << curM << " metal." << endl;
         totW += curW;
         totB += curB;
         totM += curM;
      }

      // Output totals for the month
      cout << "Collected a total of " << totW << " wood, " << totB << " brick, " << "and " << totM << " metal!" << endl << endl;
   }
}
